README.md

```diff
@@ -20,12 +26,276 @@ $ python setup.py install # install wallet-sdk-py
+
+```python
+>>> from api.wallet import WalletClient
+>>> url = "http://127.0.0.1:9143"
+>>> apikey = "pWEzB4yMM1518346407"
+>>> cert_path = "/usr/local/lib/python2.7/site-packages/py_common-1.5.0-py2.7.egg/cryption/ecc/certs"
+>>> wallet = WalletClient(
+...     url,
+...     apikey,
+...     cert_path
+...     )
+```
+
+* At building the client configuration, the **url** and **apikey** fields must
+be set. The **url** is set to the http address of wasabi service, and the
+**apikey** is set to the API access key applied on `ChainConsole` management page.
+
+### Register wallet account
+
+After creating wallet client, you can use this client to register wallet account
+as follows:
+
+```python
+>>> header = {}
+>>> body = {
+...     "id": "",
+...     "type": "Organization",
+...     "access": "username001",
+...     "secret": "User001Pass", ##8-16 characters including upper case, lower case and digits
+...     "public_key":  {
+...         "usage": "",
+...         "key_type": "",
+...         "public_key_data": ""
+...     }
+... }
+>>> _, resp = wallet.register(header, body)
+>>> print resp
+```
+
+* `Callback-Url` in http header is optional. If you want to receive blockchain transaction
+event, you need to set `Callback-Url` in header; if not, you don't have to set it.
+
+### Create POE digital asset
+
+After creation wallet account, you can create POE asset for this account as follows:
+
+```python
+>>> from api.poe import POEClient
+>>> poe = POEClient(url, apikey, cert_path)
+>>> creator = "wallet ID"
+>>> created = "123456" ## timestamp when poe is created
+>>> privateB64 = "base64 formatted private key"
+>>> payload = {
+...     "id": "",
+...     "name": "name",
+...     "parent_id": "parent-poe-id",
+...     "owner": "owner did",
+...     "hash": "metadata-hash",
+...     "metadata": '{
+...         "address": "xxx",
+...         "telephone": "xxx",
+...         ...
+...     }'
+... }
+>>> params = {
+...     "header": {},
+...     "creator": creator,
+...     "created": created,
+...     "privateB64": privateB64,
+...     "payload": payload
+...     }
+>>> _, resp = poe.create_with_sign(**params)
+>>> print resp
+```
+
+* At creating POE asset, the **name** and **owner** fields must be set, and the
+**owner** field must be set to the wallet account ID.
+
+* At building the signature parameter, using the ed25519 private key returned
+by regiser wallet API to do ed25519 signature.
+
+### Upload POE file
+
+After creation POE, you can upload POE file for this account as follows:
+
+```python
+>>> filename = "file path"
+>>> poeid = "poeid"
+>>> _, resp = poe.upload({}, filename, poeid)
+>>> print resp
+```
+
+* `upload` API uploads the file to **Offchain** storage, and generates SHA256
+hash value for this file, then saves this hash value into blockchain.
+
+### Issue colored token using digital asset
+
+Once you have specified asset, you can use this specified asset to issue colored
```

Comment:
Once you have possessed an asset, you can use this specific asset to issue colored tokens as follows: